- Solaris specific documentation change.
- Solaris tbl(1) won't handle 'x' suffix.
- Man page category change.

--- doc/lftp.1	Tue Jun 28 02:36:24 2011
+++ doc/lftp.1	Thu Jul 14 06:55:30 2011
@@ -68,8 +68,7 @@
 the open command.
 .PP
 \fBlftp\fP can handle several file access methods - ftp, ftps, http, https,
-hftp, fish, sftp and file (https and ftps are only available when lftp is
-compiled with GNU TLS or OpenSSL library). You can specify the method to use in `open
+hftp, fish, sftp and file. You can specify the method to use in `open
 URL' command, e.g. `open http://www.us.kernel.org/pub/linux'. hftp is
 ftp-over-http-proxy protocol. It can be used automatically instead of ftp
 if ftp:proxy is set to `http://proxy[:port]'. Fish is a protocol working
@@ -164,7 +163,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 add <name> [<loc>]	T{
 add current place or given location to bookmarks and bind to given name
 T}
@@ -182,7 +181,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 stat	print cache status (default)
 on|off	turn on/off caching
 flush	flush cache
@@ -292,7 +291,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-c	continue, reget
 \-E	delete source files after successful transfer
 \-a	use ascii mode (binary is the default)
@@ -318,7 +317,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-o <lfile>	T{
 destination file name (default - basename of rfile)
 T}
@@ -343,7 +342,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-f	plain files (default)
 \-d	directories
 \-a	all types
@@ -413,7 +412,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-c	continue, reget.
 \-d	T{
 create directories the same as file names and get the files into them instead of current directory.
@@ -436,7 +435,7 @@
 .Sp
 .in +0.5i
 .TS
-l1	l	lx	.
+l1	l	l	.
 \-c,	\-\-continue	T{
 continue a mirror job if possible
 T}
@@ -620,7 +619,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-c	continue, reput
 \-d	T{
 create directories the same as in file names and put the files into them instead of current directory
@@ -668,7 +667,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-c	T{
 continue transfer. Requires \fIlfile.lftp-pget-status\fP file.
 T}
@@ -690,7 +689,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-o <rfile>	T{
 specifies remote file name (default - basename of lfile)
 T}
@@ -745,7 +744,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-q	Be quiet.
 \-v	Be verbose.
 \-Q	T{
@@ -766,7 +765,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 queue \-d 3	Delete the third item in the queue.
 queue \-m 6 4	T{
 Move the sixth item in the queue before the fourth.
@@ -829,7 +828,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-c <count>	maximum number of iterations
 \-d <delay>	delay between iterations
 \-\-while\-ok	stop when command exits with non-zero code
@@ -876,7 +875,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-a	list all settings, including default values
 \-d	list only default values, not necessary current ones
 .TE
@@ -934,7 +933,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \-O <directory>	T{
 specifies base directory where files should be placed
 T}
@@ -1130,7 +1129,7 @@
 .Sp
 .in +0.5i
 .TS
-l	lx	.
+l	l	.
 \\a	bell
 \\e	escape
 \\n	new line
@@ -1996,7 +1995,7 @@
 password is used.
 
 .SH SEE ALSO
-.BR ftpd (8),\  ftp (1)
+.BR ftpd (1M),\  ftp (1)
 .br
 RFC854 (telnet),
 RFC959 (ftp),
